dc.sh

```diff
@@ -79,5 +108,14 @@ generate_service_lists() {
 	local services_cache_file=$(get_workspace_cache_dir)/services
 	dc config --services > "$services_cache_file"
 
+	local sources_cache_file=$(get_workspace_cache_dir)/sources
+	parse_sources > $sources_cache_file
+
```

Comment:
not sure if we need to cache this,  it's not used in autocomplete and I think the parseyaml is fast